modules/oc-mirror-oci-format.adoc

```diff
@@ -5,12 +5,26 @@
 
 :_content-type: PROCEDURE
 [id="oc-mirror-oci-format_{context}"]
-= Mirroring file-based catalog Operator images in OCI format
+= Mirroring file-based OCI Operator catalogs
 
-You can use the oc-mirror plugin to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can copy local OCI images to your target mirror registry.
+You can mirror Operator images from a local file-based catalog to your target mirror registry. The catalog must be on disk and must be in the Open Container Initiative (OCI) format.
 
-:FeatureName: Using the oc-mirror plugin to mirror Operator images in OCI format
-include::snippets/technology-preview.adoc[]
+You can also mirror {product-title} releases, other Operator catalogs, and additional images from a registry along with the OCI catalog on disk to your target mirror registry.
```

Comment:
Suggestion:
While mirroring {product-title} releases,  Operator catalogs, and additional images from a registry to a partially disconnected cluster, you can include Operator catalog images from a local file-based catalog on disk. 
The local catalog and its contents will be mirrored to your target mirror registry, based on the filtering information in the imagesetconfig configuration file.

Reply:
Added with minor tweaks. Should I remove the beginning sentence of this file with this replacement, or keep it?

If I need to remove it, I'd need somewhere in there to mention and spell out "Open Container Initiative (OCI)"

Reply:
I see what you mean... Does it make better sense if we just switch order for L10 and L12?

Reply:
@sherine-k I ended up just getting rid of that first sentence and moved just the piece about OCI format to the end of the first sentence paragraph. Let me know if that works!